src/Language/PureScript/Parser/Declarations.hs

```diff
@@ -231,6 +237,7 @@ parseLocalDeclaration :: TokenParser Declaration
 parseLocalDeclaration = positioned (P.choice
                    [ parseTypeDeclaration
                    , parseValueDeclaration
+                   , parseBoundValueDeclaration
```

Comment:
Oh, sorry I mistook. The first pattern should be able to be n-ary, because the following pattern is also correct.

```purescript
let X a = X 10
in
```